src/librustdoc/lib.rs

```diff
@@ -269,6 +269,26 @@ fn opts() -> Vec<RustcOptGroup> {
                 "sort modules by where they appear in the program, rather than alphabetically",
             )
         }),
+        unstable("default-theme", |o| {
+            o.optopt(
+                "",
+                "default-theme",
+                "Set the default theme.  THEME should be the theme name, generally lowercase. \
```

Comment:
The usage of the double whitespace is bugging me a lot. Is there a reason behind it?